client/containers/Login/logout.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { LogOutUser } from '../../store/actions/auth/login';
+
+export class LogOut extends Component {
```

Comment:
Preferably, component should be named Logout
```suggestion
export class Logout extends Component {
```
https://www.computerhope.com/jargon/s/signoff.htm